_rules/block-collapsible-3e12e1.md

```diff
@@ -0,0 +1,489 @@
+---
+id: 3e12e1
+name: Block of content is expandable and collapsible
+rule_type: atomic
+description: |
+  This rule checks that repeated blocks of content are expandable and collapsible
+accessibility_requirements:
+  wcag-technique:SCR28: # Using an expandable and collapsible menu to bypass block of content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation 1
+
+There exists a [semantic segmentation][] of the test target such that for each [block][], B, in this [segmentation] which is before the [main block of content][] and contains at least one [block of repeated content][], there exists a [visible][] [instrument][] to make this [block][] (B) not [visible][].
```

Comment:
The SC does not require that the instrument should be visible and in addition, this SC is a related screen reader user who doesn't need any visible instrument as long as it is keyboard accessible such as focusable. I personally conducted interviews with a screenreader user who even does not have a monitor or screen. 

Well, here we need to define criteria if you agree, for example in how many tabs from the start of the page this instrument which must be keyboard accessible when getting focus should be positioned such as within the first three tabs.

Reply:
This SC is related to more users than just screenreader users. Keyboard users that don't want to tab past a large main menu every time they navigate to a different page would also be helped by an instrument such as this and I would say that if such a mechanism is not visible to such users it is not "available" and thus should fail the SC.

Reply:
More thinking about this, and I tend to agree with you (maybe not for the same reasons).

---
If there is a menu labelled "hide blocks" which, when open, allow to toggle on and off each block, I would say that this satisfy 2.4.1, but it wouldn't pass this rule because the instruments are the items in the menu, hidden by default (the menu itself is not the instrument since its activation only opens it but does not toggle the blocks).

So, it is probably better to require the instrument for toggling visibility to be visible when focused, but not necessarily always.


@ShadowBB, @carlosapaduarte WDYT?